### PR TITLE
add NAMD FEP test data

### PR DESCRIPTION
**Pull request by vtlim**

### CODE REVIEW

File: src/alchemtest/namd/tyr2ala/descr.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+=======================================================================
+ NAMD: free energy of tyrosine to alanine mutation in aqueous solution
+=======================================================================
+
```
Comment (orbeckst) [inline src/alchemtest/namd/tyr2ala/descr.rst]: @vtlim please add details to the description. See the existing `descr.rst` files such as  [amber/simplesolvated/descr.rst](https://github.com/alchemistry/alchemtest/blob/master/src/alchemtest/amber/simplesolvated/descr.rst).

Most importantly: it needs to have a *license* and the  *donor* should be who can grant the license.

**Comment by orbeckst** [comment]
Sorry it took me so long to review. You can always ask for a review in a comment (use e.g. @orbeckst to notify me) or *request a reviewer*, using the *Reviewer* widget in the upper right hand corner of the PR.

File: src/alchemtest/namd/tyr2ala/descr.rst (hunk 1 above)
Comment (orbeckst) [reply src/alchemtest/namd/tyr2ala/descr.rst]: According to the discussion https://github.com/alchemistry/alchemtest/issues/17#issuecomment-349471034 , the donor should be JC Gumbart and the license is CC0, or has anything changed?

If the source is an existing file (NAMD tutorial??) then please also add to the text where you got the files from.

**Comment by orbeckst** [comment]
@vtlim can you please address the remaining minor comments? Then I can merge. Thanks!

File: src/alchemtest/namd/tyr2ala/descr.rst (hunk 1 above)
Comment (vtlim) [reply src/alchemtest/namd/tyr2ala/descr.rst]: @orbeckst -- done. The donor is JC Gumbart, yes. Is there place where the input files should go?

Comment (orbeckst) [reply src/alchemtest/namd/tyr2ala/descr.rst]: Thanks for adding the meta data.

> Is there place where the input files should go?

We don't store input files here. If there's a web page where people could go to or if you want to put the files on zenodo or figshare or GitHub then that URL could be added to the text. 

Is there a URL you want to add right now?

**Comment by vtlim** [comment]
@orbeckst -- I don't have a URL to add at the moment. JC Gumbart also sent the NAMD configuration files to Michael but those haven't been uploaded.

**Comment by orbeckst** [comment]
Ok, thanks, merging now.

**Pull request merged**